zad16.py

```diff
@@ -0,0 +1,39 @@
+from meszstack import meszgen
+
+
+def wyraz(s1, s2=""):
+
+    # counts vowels in string
+    def vowels(text):
+        count = 0
+        for letter in text:
+            for a in "aeiouy":
+                if letter is a:
+                    count += 1
+        return count
```

Comment:
Weź to wyciągnij bo nie ma po co robić to nested

Reply:
Fixed 4b8c82dd70673d7ec5ec7a2e2e3a97dd2287bd54